include/benchmark/benchmark.h

```diff
@@ -429,6 +429,46 @@ struct Statistics {
       : name_(name), compute_(compute) {}
 };
 
+// Converts whatever the stored manual time is to
+// seconds, which is used to estimate iterations
+// and similar
+typedef double(BenchmarkTimeCostFunc)(double);
+
+struct BenchmarkTime {
+  BenchmarkTimeCostFunc* to_cost_in_seconds_;
+
+  static double SecondsCost(double measurement) { return measurement; }
```

Comment:
this should be part of the public API, but i don't think the rest of the class needs to be.